.cloudbuild/prombench.yaml

```diff
@@ -0,0 +1,487 @@
+steps:
+  - id: "Verifying GitHub comment and user permissions"
+    name: bash
+    env:
+    - '_COMMENTER=$_COMMENTER'
+    - '_COMMENTER_ROLE=$_COMMENTER_ROLE'
+    - '_GIT_REPO=$_GIT_REPO'
+    - '_PR_COMMENT_ACTION=$_PR_COMMENT_ACTION'
+    - '_PR_COMMENT_TEXT=$_PR_COMMENT_TEXT'
+    - '_PR_NUMBER=$_PR_NUMBER'
+    secretEnv: ['GH_PAT']
```

Comment:
What's PAT?

Reply:
Personal Access Token?

Reply:
It's GitHub bot's Personal Access Token (to authenticate to GitHub).